docs/how_to_dev_frontend_en.md

```diff
@@ -0,0 +1,105 @@
+# VisualDL Web App Develop
+
+VisualDL has two major components. One is the Python/C++ SDK that logs the matrix during training,
+the other is the web-app that display visualized training data.
+
+The web-app reads the extract data and draw scalars graph or histograms graphs.
+The web-app is where VisualDL transforms the code data into graphs so users can visualize the training progress and make a better network design evaluation.
+
+This article will go over the basic web-app architecture and development steps.
+
+## Web App Overview
+The VisualDL web-app home page is constructed by three major Javascript files
+1. manifest.js: The manifest configures basic settings  
+1. vendor.js: The vendor.js stores all the 3rd party libraries.
+1. index.js: The index.js contains all the graph algorithms. All navigations, setting panels are implemented here.
```

Comment:
Why do you say all the magic happens here? There are so many component files and dirs. Why not list them?

Reply:
the index.js is a packaged file. I can't explain the purpose of each component yet. Like you said, there are many components. This article is a start/placeholder to kick off the documentation process. Please do feel free to add the detail later.

Reply:
Yes, so indeed, you shouldn't mention `index.js`. Its like a `.pyc` file, its built - you need to refer to the source files and directories (even if you aren't going into them)